### PR TITLE
Update terminal tutorial

### PR DESCRIPTION
**Pull request by dmytrovoytko**
According to #2828
FYI @TenTraicion

### CODE REVIEW

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@dmytrovoytko, sorry for the late merge. I just got approval from the community maintainers. Good work and looking forward to your future contributions. Thanks.

**Comment by dmytrovoytko** [comment]
Awesome, thank you!
I'm glad maintainers checked the updated manual and approved it, as it's so easy to miss something in a quite big doc when doing it alone. That's where the power of community comes into play!

Going to translate it to Ukrainian soon.

**Pull request merged**
